grammars/asciidoc.cson

```diff
@@ -84,6 +206,32 @@
     ]
   }
 
+  # Matches explicit paragraph styles
+  #
+  # Examples
+  #
+  #   [normal]
+  #   This is a normal paragraph.
+  {
+    'match': '^\\[((normal|literal|listing|TIP|NOTE|IMPORTANT|WARNING|CAUTION|partintro|comment|example|sidebar|source|music|latex|graphviz))\\]$'
```

Comment:
If you agree with [this comment](https://github.com/asciidoctor/atom-language-asciidoc/pull/17/files#r36923465), we could generalize this pattern to match any text within the square brackets.